src/nupic/os/Regex.cpp

```diff
@@ -46,7 +47,14 @@ namespace nupic
       exactRegExp += re;
       if (re[re.length()-1] != '$') 
         exactRegExp += '$';
-      
+
+#if defined(NTA_OS_WINDOWS)
```

Comment:
We did have PCRE working on Windows. But at the time there were comments about not wanting to pollute the repo with an OS specific library. We moved to std::regex to be able to remove that library.